content/shared/influxdb3-query-guides/snapshots/snowflake.md

```diff
@@ -0,0 +1,140 @@
+# Integrate with Snowflake using Apache Iceberg
+
+
+Export time-series data snapshots from InfluxDB into Apache Iceberg format.
+Integrate data with Snowflake and other Iceberg-compatible tools without the need for complex ETL processes.
+
+### Key Benefits
+
+- **Efficient data access**: Query InfluxDB data directly from Snowflake.
```

Comment:
```suggestion
- **Efficient data access**: Query your data directly from Snowflake.
```

We avoid saying "InfluxDB data", because it's the customer's data.